core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -304,6 +304,14 @@ copy_init:
 	bgt	copy_init
 #endif
 
+	/* Clear .bss */
+	ldr	r0, =__bss_start
+	ldr	r1, =__bss_end
+	mov	r2, #0
+clear_bss:
+	strb	r2, [r0], #1
```

Comment:
This function could also be optimized by using the `stm` instruction, but I think it's OK to take such optimizations later.